### PR TITLE
1020 add tests and fixes

### PR DESCRIPTION
**Pull request by yaroslav-fedyshyn-nordwhale**
# Description

Add tests for Hanuka endpoint.
Apply fixes required in the main PR.

Here is the link to the main (previous) PR - https://github.com/GoodDollar/GoodServer/pull/131.

About https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/1020

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request ( for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

**Comment by AlexeyKosinski** [comment]
image

**Comment by yaroslav-fedyshyn-nordwhale** [comment]
@AlexeyKosinski @sirpy 
I missed that last comment yesterday. I just pushed the new commit with the changes

File: src/server/verification/verificationAPI.js [hunk 1]
```diff
@@ -466,6 +468,11 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
           log.error('Bonuses charge failed', e.message, e, user)
 
           fail()
+
+          return res.status(400).json({
```
Comment (sirpy) [inline src/server/verification/verificationAPI.js]: you need to check a response was not already sent by ontxhash

Comment (yaroslav-fedyshyn-nordwhale) [reply src/server/verification/verificationAPI.js]: done

**Pull request merged**